### ISSUE TITLE
[Search]: Home button should move search cursor to the beginning of Input when focused

### ISSUE THREAD

**Issue by electrostaticfleece** [carbon-design-system/carbon#6456]
## What package(s) are you using?

- [X] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.

When using a Windows-based OS the `Home` key should move the cursor to the beginning of the input for the Search component when focused. However, when pressing the `Home` key using the [Search component](https://react.carbondesignsystem.com/?path=/story/search--default) this does not occur. 

Also, I'm actually not sure if this is a bug or should be a feature request. Please relabel or let me know if I need to open a separate issue for this. 

> Is this issue related to a specific component?

Search

> What did you expect to happen? What happened instead? What would you like to

See the description above.

> What browser are you working in?

I actually don't have a reference for this, as I use Mac OS, but this was originally reported by @gptt916 and he should be able to provide you with further details if needed.

> What version of the Carbon Design System are you using?

Currently on `10.11.2`, but should be occurring on latest as well

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

IBM Cloud Docs

## Steps to reproduce the issue

1. Visit https://react.carbondesignsystem.com/?path=/story/search--default
2. Click on the search bar and type in some input
3. Press the home key using Windows OS
4. The cursor does not move to the beginning of the input like desired.

**Comment by emyarod** [comment]
I am unable to reproduce this issue in the component on Windows or macOS, but if you can provide any more information on how to replicate the bug then it would be very helpful. are there keyboard event handlers in your application that might be affecting this behavior?

**Comment by electrostaticfleece** [comment]
Apologies. This has been resolved and can be closed.
